app/models/book.rb

```diff
@@ -0,0 +1,7 @@
+class Book < ApplicationRecord
+  validates :title, presence: true
+  validates :author, presence: true
+  validates :genre, presence: true
+  validates :publisher, presence: true
+  validates :year, presence: true
+end
```

Comment:
And the `image` field?
You can write same validations on the same line:
```ruby
validates :author, :year, ...,presence: true
```

Reply:
I keep forgetting Book has an image field. Sorry, I will update it now.